packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -71,29 +70,25 @@ export const Table = styled(HTMLTable)<{ $showFooter: boolean }>`
 
         th:first-child {
             border-top: none !important;
-            border-bottom: none !important;
         }
 
         th {
-            border-top: none !important;
-            border-bottom: none !important;
-            box-shadow: inset 0 1px 0 #dcdcdd, inset 0 -1px 0 #dcdcdd,
-                inset 1px 0 0 0 rgb(17 20 24 / 15%) !important;
+            border-top: 1px solid #d3d3d3 !important;
+            border-bottom: 1px solid #d3d3d3 !important;
         }
     }
 
     .sticky-column {
         position: sticky !important;
         left: 1px;
         z-index: 1;
-        background-color: white;
         word-break: break-word;
     }
-    .first-sticky-column {
-        box-shadow: lightgray -1px 0px 0px 0px, lightgray 0px 1px 0px 0px inset !important;
-    }
+
     .last-sticky-column {
-        border-right: 2px solid darkgray;
+        background-color: rgb(222, 222, 222);
+        border-right: 2px solid rgb(245, 245, 245);
+        border-left: 2px solid rgb(245, 245, 245);
```

Comment:
we might need to update this with something much liter. can you use colours provided by the blueprint library?